theme/int/views/authentication/home.hbs

```diff
@@ -29,15 +29,143 @@
+                            <div class="text-xs-center text-md-right">
+                                <img class="mb-3 mb-md-0 section-img" src="/images/home/advantage_2.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="col-md-8 pull-md-4">
+                            <h5 class="heading font-weight-bold ">Einfach in der Anwendung –   jederzeit und überall</h5>
+                            <p class="mb-2">Die HPI Schul-Cloud International ist sicher, transparent und kann ohne lange Einarbeitung genutzt werden – zeitunabhängig und mit dem eigenen Handy, Tablet oder Computer.</p>
+                        </div>
+                    </div>
+
+                    <div class="row flex-container mb-3">
+                        <div class="col-sm-12 col-md-4 text-xs-center">
+                            <img class="mb-3 mb-md-0 section-img" src="/images/home/advantage_3.svg" alt="">
+                        </div>
+                        <div class="col-sm-12 col-md-8">
+                            <h5 class="heading font-weight-bold ">Datenschutzkonform und sicher</h5>
+                            <p class="mb-2">Die HPI Schul-Cloud International wird in und für Deutschland entwickelt. Datenschutzkonformer Zugriff auf Lernsoftware, sowie Datensouveränität, sind gesichert.</p>
+                            <link><a class="font-weight-bold" href="https://schul-cloud.org/mint-ec/06-datenschutz-und-urheberrecht">Mehr erfahren <i class="fa fa-angle-double-right font-weight-bold"></i></a>
+                        </div>
+                    </div>
+
+                    <div class="row flex-container mb-3">
+                        <div class="col-md-4 push-md-8">
+                            <div class="text-xs-center text-md-right">
+                                <img class="mb-3 mb-md-0 section-img" src="/images/home/advantage_4.svg" alt="">
+                            </div>
+                        </div>
+                        <div class="col-md-8 pull-md-4">
+                            <h5 class="heading font-weight-bold">Digitale Bildung partnerschaftlich gestalten </h5>
+                            <p class="mb-2">Gemeinsam mit unseren starken Partner:innen, den deutschen Auslandsschulen, gefördert durch das auswärtige Amt, wird die HPI Schul-Cloud International weiterentwickelt.</p>
+                            <link><a class="font-weight-bold" href="https://blog.hpi-schul-cloud.de/about">Mehr erfahren <i class="fa fa-angle-double-right font-weight-bold"></i></a>
+                        </div>
+                    </div>
+
+                    <section class="section section-tilt">
+                        <div class="container container-content content-width-max">
+                            <div style="text-align: center;">
+                                <h2 class="h3 section-title center" style="max-width: 20ch; margin: 0 auto;">
```

Comment:
more inline CSS

Reply:
see my other comments ➙ just copied the production CMS code.